tools/fast/series/series_sendfilecoindefaults.go

```diff
@@ -0,0 +1,19 @@
+package series
+
+import (
+	"context"
+
+	"github.com/filecoin-project/go-filecoin/address"
+	"github.com/filecoin-project/go-filecoin/tools/fast"
+)
+
+// SendFilecoinDefaults sends the `value` amount of fil from the default wallet
+// address of the `from` node to the `to` node's default wallet.
+func SendFilecoinDefaults(ctx context.Context, from, to *fast.Filecoin, value int) error {
```

Comment:
note this can only sent integer number of fil.

Reply:
This is for API consistency with the other series used in this function which was added prior to our understanding of how `message send` actually worked. This will be updated later.